examples/blender/blender.py

```diff
@@ -79,9 +81,21 @@ async def worker(ctx: WorkContext, tasks):
                 raise
 
             if show_usage:
-                print(f" --- {ctx.provider_name} USAGE: {await ctx.get_usage()}")
-                print(f" --- {ctx.provider_name} STATE: {await ctx.get_state()}")
-                print(f" --- {ctx.provider_name}  COST: {await ctx.get_cost()}")
+                print(
+                    f"{TEXT_COLOR_MAGENTA}"
+                    f" --- {ctx.provider_name} STATE: {await ctx.get_raw_state()}"
+                    f"{TEXT_COLOR_DEFAULT}"
+                )
+                print(
+                    f"{TEXT_COLOR_MAGENTA}"
+                    f" --- {ctx.provider_name} USAGE: {format_usage(await ctx.get_usage())}"
+                    f"{TEXT_COLOR_DEFAULT}"
+                )
+                print(
+                    f"{TEXT_COLOR_MAGENTA}"
+                    f" --- {ctx.provider_name}  COST: {await ctx.get_cost()}"
+                    f"{TEXT_COLOR_DEFAULT}"
+                )
```

Comment:
Since we're adding common code to `examples/utils.py` (namely the `format_usage()` function), perhaps we can place this whole block of code there? It's duplicated in `simple_service.py`.

Reply:
nah, it's not the same... and I would still like to keep those calls as part of the example